rosbag2_performance/rosbag2_performance_benchmarking/test/benchmark_test.py

```diff
@@ -0,0 +1,418 @@
+# Copyright 2021 Apex.AI, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import csv
+import fnmatch
+import os
+import pathlib
+import statistics
+import unittest
+
+import ament_index_python
+
+import launch
+import launch.actions
+import launch_testing
+
+import yaml
+
+# Postprocess and PostprocessStorageConfig classes have been adapted from report_gen.py script
```

Comment:
@MichaelOrlov I would a TODO comment here explaining this can be refactored and move the helper functions out of the benchmark test.

Reply:
Added